ansible-runner/context/_build/requirements.txt

```diff
@@ -1,2 +1 @@
-ansible
-numpy
\ No newline at end of file
+ansible==5.0.1
```

Comment:
Do we need this dependency still?

Reply:
Can probably remove the requirements. Let me do that